packages/mui-joy/src/styles/extendTheme.ts

```diff
@@ -379,45 +310,41 @@ export default function extendTheme(themeOptions?: CssVarsThemeOptions): Theme {
   const fontFamilyFallback =
     '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"';
   const fontFamily = {
-    body: `"Public Sans", ${getCssVar('fontFamily-fallback', fontFamilyFallback)}`,
-    display: `"Public Sans", ${getCssVar('fontFamily-fallback', fontFamilyFallback)}`,
+    body: `"Inter", ${getCssVar('fontFamily-fallback', fontFamilyFallback)}`,
+    display: `"Inter", ${getCssVar('fontFamily-fallback', fontFamilyFallback)}`,
     code: 'Source Code Pro,ui-monospace,SFMono-Regular,Menlo,Monaco,Consolas,Liberation Mono,Courier New,monospace',
     fallback: fontFamilyFallback,
     ...scalesInput.fontFamily,
   };
 
   const fontWeight = {
-    xs: 200,
-    sm: 300,
+    xs: 300,
+    sm: 400,
     md: 500,
     lg: 600,
     xl: 700,
-    xl2: 800,
-    xl3: 900,
     ...scalesInput.fontWeight,
   };
 
   const fontSize = {
-    xs3: '0.5rem',
-    xs2: '0.625rem',
-    xs: '0.75rem',
-    sm: '0.875rem',
-    md: '1rem',
-    lg: '1.125rem',
-    xl: '1.25rem',
-    xl2: '1.5rem',
-    xl3: '1.875rem',
-    xl4: '2.25rem',
-    xl5: '3rem',
-    xl6: '3.75rem',
-    xl7: '4.5rem',
+    xs: '0.75rem', // 12px
+    sm: '0.875rem', // 14px
+    md: '1rem', //16px
+    lg: '1.25rem', // 20px
+    xl:'1.5rem', // 24px
+    xl2:'2rem', // 32px
+    xl3:'2.5rem', // 40px
+    xl4:'3.5rem', // 56px
+    xl5:'4rem', // 64px
+
     ...scalesInput.fontSize,
   };
 
   const lineHeight = {
```

Comment:
I've changed the lineHeight to rem, don't know if this is the best way to do it, though. Maybe as a multiplier it'd be more scalable, but for the default theme I reckon it was easier to keep the consistency with rem instead.

Reply:
I have seen a good number of pushback on anything else than unitless for the line height, https://meyerweb.com/eric/thoughts/2006/02/08/unitless-line-heights/, https://allthingssmitty.com/2017/01/30/nope-nope-nope-line-height-is-unitless/. Material UI and Bootstrap are unitless.